packages/@aws-cdk/aws-ecs-patterns/test/ec2/l3s-v2.test.ts

```diff
@@ -1081,6 +1083,45 @@ describe('When Network Load Balancer', () => {
       PlacementStrategies: [{ Field: 'instanceId', Type: 'spread' }, { Field: 'cpu', Type: 'binpack' }, { Type: 'random' }],
     });
 
+    // ECS Exec
+    Template.fromStack(stack).hasResourceProperties('AWS::IAM::Policy', {
```

Comment:
Can you put this in a separate test called something like "setting enableExecutdCommand to true creates the right IAM permissions"?